File: PHP/Euler02/Euler02.php

<?php

// placeholder


$finalanswer = 0;

$tic = microtime(true);


$toc = microtime(true);
$raw_duration = ($toc - $tic)/60;
$elapsed = number_format((float) $raw_duration, 10);

echo "Answer:\t".$finalanswer."\nTook:\t".$elapsed;

?>

File: Pascal/Euler02/Euler02.pas
{ Placeholder }

// Placeholder

program Euler02;

	uses sysutils;

	var
		finalanswer : longint = 0;
		tic, toc, duration: TDateTime;

begin

	tic := Now;

	
 	toc := Now;
 	duration := toc - tic;

	writeln('Answer:	', finalanswer);
	writeln('Took:	', duration :0:6, ' seconds');

end.